### ISSUE TITLE
System UI completely breaks when setting certain images as wallpaper

### ISSUE THREAD

**Issue by MrSoke** [SimpleMobileTools/Simple-Gallery#969]
This is a big problem and it seems to only happen with Simple Gallery.

I use Simple Gallery as the app to set my wallpaper, it never gave me much problem until I tried to set a particular image as a wallpaper. When I did, I got the error message telling me the System UI stopped working, I clicked on "reopen" and it wouldn't work anymore. The screen turned black and the lock screen appeared (without any wallpaper by the way) and disappeared intermittently. It appeared for about half a second and then disappears for about 1 second, then appears and disappears again and so about 10 times more. After that I get an error message telling me something like "there is an internal error and the device will be unstable until you do a factory reset" (not exact words because my phone is in spanish), I click ok and the screen just stays black, nothing happens and nothing responds. If I restart the phone the same thing happens again and I'm unable to do anything. Rebooting in safe mode didn't help at all, the only way I could solve this was by doing a hard reset and wiping all my data with a factory reset.

I decided to see what happened and had to do multiple factory resets in the process. I found out this happens with two particular images I have, the only thing I can think of that they have in common is the fact that they were both drawings I made in Krita and they're both in a vertical format (although other images in vertical format and other images also drawn in Krita worked fine). I tried setting these exact same images as my wallpaper using both the default google wallpapers app and google photos and it worked perfectly fine with both apps, the System UI only breaks when I try to set those pictures as wallpaper with Simple Gallery, this is what makes me believe this is a problem with the app and not a problem with my phone.

I am using a Moto G6 plus with Android 8.0.0. In case it helps in any way, the images in question were both made with Krita in Linux, they are .png files and their resolutions are 1152x2048 and 935x1277.

**Comment by MrSoke** [comment]
No, it's not rooted and it has the version of android that comes preinstalled in the device

**Comment by tibbi** [comment]
I dont think it was an apps issue, anyway, many things changed since this already
